sidekick/lib/src/init/init_command.dart

```diff
@@ -67,6 +67,15 @@ class InitCommand extends Command {
     if (!isValidCliName(cliName)) {
       throw invalidCliNameErrorMessage;
     }
+    bool cliNameCollides = false;
+
+    which(cliName).paths.forEach((element) {
+      cliNameCollides = !element.contains('.sidekick');
```

Comment:
This actually only checks the `paths.last`, all previous iterations are overwritten